src/main/java/cubicchunks/worldgen/generator/ICubeGenerator.java

```diff
@@ -36,12 +36,12 @@
 
 public interface ICubeGenerator {
 
-	static final Box RECOMMENDED_POPULATOR_REQUIREMENT = new Box(
+	Box RECOMMENDED_POPULATOR_REQUIREMENT = new Box(
 				-1, -2, -1, // give an extra 16 blocks virtical buffer for things like jungle trees
 				 0,  0,  0
 			);
 
-	static final Box NO_POPULATOR_REQUIREMENT = new Box(
+	Box NO_POPULATOR_REQUIREMENT = new Box(
```

Comment:
Fields in interfaces are static and final by default... (IDEA would not stop bothering me about them :P)